c2corg_api/views/validation.py

```diff
@@ -57,6 +57,28 @@ def validate_lang_(lang, request):
             request.errors.add('querystring', 'lang', 'invalid lang')
 
 
+def validate_pref_lang_(lang, request):
+    """Checks given langs are one of the available langs.
+    """
+    if lang is not None:
+        validated = []
+        if ',' in lang:
+            langs = lang.split(',')
+            for lang in langs:
+                if lang in default_langs:
+                    validated.append(lang)
+        elif lang in default_langs:
+            validated.append(lang)
+
+        if validated:
+            request.validated['lang'] = validated
+
+        # if lang in default_langs:
+        #     request.validated['lang'] = lang
+        else:
+            request.errors.add('querystring', 'lang', 'invalid lang')
```

Comment:
This means that if you pass an incorrect lang in the comma-separated list, (eg. ``fr,de,ru``), the incorrect lang will be ignored but no error will be thrown, right?
Shouldn't it throw an error whenever an invalid lang is detected?